liquibase-standard/src/main/java/liquibase/sqlgenerator/core/RawParameterizedSqlGenerator.java

```diff
@@ -13,7 +13,7 @@ public class RawParameterizedSqlGenerator extends AbstractSqlGenerator<RawParame
     public ValidationErrors validate(RawParameterizedSqlStatement statement, Database database, SqlGeneratorChain<RawParameterizedSqlStatement> sqlGeneratorChain) {
         ValidationErrors validationErrors = new ValidationErrors();
         validationErrors.checkRequiredField("sql", statement.getSql());
-        validationErrors.checkRequiredField("parameters", statement.getParameters(), false);
+        validationErrors.checkRequiredField("parameters", statement.getParameters(), true);
```

Comment:
This is a notable change. @MalloD12 can you explain it and get feedback from the rest of the team on it?

Reply:
So.. why it changed?

Reply:
Because now we can use it without parameters, got it!